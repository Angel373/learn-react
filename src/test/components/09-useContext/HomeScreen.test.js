import { mount } from "enzyme"
import React from 'react'
import {HomeScreen} from '../../../components/09-useContext/HomeScreen'
import { UserContext } from "../../../components/09-useContext/UserContext"
describe('Pruebas en el componente <HomeScreen/>', () => {
    test('debe de mostrarse correctamente', () => {
        
        const user = {
            name: 'jhon',
            email: 'jhon@gamil.com'
        }
        const wrapper = mount(
            <UserContext.Provider value={{
                user
            }}>
                
                <HomeScreen/>
            </UserContext.Provider>

        )
        expect(wrapper).toMatchSnapshot();
    })
    
})
