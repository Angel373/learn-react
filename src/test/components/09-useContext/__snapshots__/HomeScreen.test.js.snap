// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HomeScreen/> debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "jhon",
   "email": "jhon@gamil.com"
}
  </pre>
</div>
`;
